cmd/lncli/cmd_pay.go

```diff
@@ -147,14 +147,15 @@ var sendPaymentCommand = cli.Command{
 	If payment isn't manually specified, then only a payment request needs
 	to be passed using the --pay_req argument.
 
-	If the payment *is* manually specified, then all four alternative
+	If the payment *is* manually specified, then all five alternative
```

Comment:
for the wording here, i think we want to say that the first four are required, and that `pay_addr` may be required by the receiver. it is still optional as there are still nodes that don't signal it in their invoice. additionally, keysend does not need a `pay_addr` since it's built on the legacy format

Reply:
The wordings are changed, might be a bit long...

Reply:
Looks good now!